.gitattributes

```diff
@@ -26,3 +26,11 @@ Dockerfile text
 
 # Explicitly denote all files that are truly binary and should not be modified.
 # *.jpg binary
+
+# Declare files that should be ignored when creating an archive of the
+# git repository
+.gitignore export-ignore
+.gitattributes export-ignore
+gradlew export-ignore
```

Comment:
I thought the concern was only about gradle/wrapper/gradle-wrapper.jar ?

Reply:
Is there a reason to leave a partial gradlew install here? Is it self-re-installing?

Reply:
It isn't self installing. I agree with Kenn that keeping some of the build systems files would be strange.